### ISSUE TITLE
Modify Search Bar - fix move and down for 2.1.42+

### ISSUE THREAD

**Issue by fUpDWfYS** [ijgnd/anki__BetterSearch#20]
Hello! I'm on a Windows version of Anki 2.1.42 and it seems that the "down" and "false" options are not working in the config to modify the search bar. When changing from one to the other nothing changes. Additionally, the Search Dialog & Filter buttons seem to have disappeared.

The "multiline" option seems to work but the Search Dialog & Filter buttons are still gone.

Thank you for a great add-on!

**Comment by ijgnd** [comment]
Thanks for this report.

2.1.41 removed some buttons and functions so I disabled some functions (that I guessed few people would use). 

The browser will soon be changed again, see [here](https://forums.ankiweb.net/t/feedback-wanted-for-new-feature-notes-view/8633) which has been merged recently, see [here](https://github.com/ankitects/anki/pull/1100). So I wanted to save some time and directly adjust all my browser ui related add-on to this upcoming latest version.
